lib/Partitioner/Partitioner.cpp

```diff
@@ -567,7 +567,7 @@ void Partitioner::doPartitioning(Function *F, NodeToFunctionMap &mapping) {
   } else if (saturateHost_) {
     // Attempt to saturate the host. Passing in the count of logical devices.
     // Since logicalId starts at 0 we add one.
```

Comment:
comment is probably stale now, since we do not explicitly add 1 here.

Reply:
Will fix the comment.